nss-cohort-36/bangazon-workforce-mgt-virtual-magicians
# Employee adding 1

# Description
This PR addresses the feature to add an employee as well as placing them in their respective department.
Closes #18 
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Go to the following link `http://localhost:8000/employee/form` and add an employee! You should see the new employee with the department they belong too!
1. Ensure that connection.py contains the appropriate absolute path for your machine to test locally
1. `git checkout master`
1. `git fetch --all`
1. `git checkout Employee-adding-1`
1. [Enter specific testing instructions here to approve]
# Checklist:
